pkg/api/types.go

```diff
@@ -1110,6 +1116,33 @@ type ScaleIOVolumeSource struct {
 	ReadOnly bool
 }
 
+// Represents a StorageOS persistent volume resource.
+type StorageOSVolumeSource struct {
+	// VolumeName is the human-readable name of the StorageOS volume.  Volume
+	// names are only unique within a namespace.
+	VolumeName string
+	// Namespace specifies the scope of the volume name within StorageOS.  If no
+	// namespace is specified, the namespace of the pod will be used.  Set to
+	// "default" if you are not using namespaces within StorageOS.
+	// +optional
+	Namespace string
```

Comment:
We use namespaces within StorageOS too, so by default it will use the Pod's namespace but you can force it not to use StorageOS namespace's at all by setting to "default".  You can also specify a different StorageOS namespace here.  Normally the StorageOS namespaces will mirror Kubernetes'.